lib/fpm/package.rb

```diff
@@ -249,8 +257,20 @@ def output(path)
                                   "creating #{self.type} packages")
   end # def output
 
+  def temporary_directory(path)
+    if attributes[:source_date_epoch].nil?
+      return Stud::Temporary.directory("package-#{type}-staging")
+    else
+      # Choose a predictable build path; see https://reproducible-builds.org/docs/build-path/
+      root = ENV["TMP"] || ENV["TMPDIR"] || ENV["TEMP"] || "/tmp"
```

Comment:
I may not be understanding this scenario, btw. Temporary directories, work directories, etc, are only need to exist during `fpm`'s run -- when `fpm` exits, they should be automatically removed.

Can you help me understand what about temporary paths is important to this 'repeatable build' goal?